README.md

```diff
@@ -177,18 +176,3 @@ When running the Clojure tests (`lein test :unit`, `lein test :compliance`), you
 `project.clj` file under the `:test` profile. For ClojureScript, you'll currently need to modify the level in the
 `test.config` namespace before building, as this file is sourced into the cljs tests.
 
-## License
-
-Copyright 2022 Johan Fylling, Anders Eknert
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-       http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
```

Comment:
I think the `LICENSE` file is just a template; should it be updated with years and names?

Reply:
IANAL, but I think we can/should leave the LICENSE file alone. If we want to make a copyright claim somewhere it should probably be for "the Jarl contributors" given the liberal license. I thought we did something as such in OPA, but I can't find it now 🙁